Detections/AzureDevOps/AzDOAdminGroupAdditions.yaml

```diff
@@ -7,7 +7,7 @@ requiredDataConnectors:
     dataTypes:
       - AzureDevOpsAuditing
 severity: High
```

Comment:
suggest changing severity to medium. 
We only have very less high severity detections and this can be downgraded to medium based on the criteria compared with others

Reply:
done